utils.py

```diff
@@ -0,0 +1,156 @@
+# more imports than we need
+import pandas as pd
+import matplotlib.pyplot as plt
+import requests
+import json
+import csv
+import io
+import numpy as np
+import pydantic
+from pydantic import BaseModel
+from datetime import datetime as dt
+from datetime import timedelta
+from typing import Tuple
+import fetch
+
+def all_submissions_df(state: str # (str: "ohio" or "michigan", i.e.)
```

Comment:
This information/comment should be elsewhere, not half-way in the function arguments. Sometimes, concise example invocations are included in the module-level or even function-level docstring (i.e. https://google.github.io/styleguide/pyguide.html#2134-decision). Another way you could achieve this would be by making this argument optional:
```python
def all_submissions_df(state: str = "michigan") -> pd.DataFrame:
```